src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -97,6 +103,32 @@ def cli(
     print(json.dumps(aligned_gisaid_dump))
 
 
+def dump_yaml_template(
+    phylo_run_id: int,
+    builds_file_fh: io.TextIOWrapper,
+):
+    interface: SqlAlchemyInterface = init_db(get_db_uri(Config()))
+
+    num_sequences: int = 0
+    num_included_samples: int = 0
+
+    with session_scope(interface) as session:
+        phylo_run = get_phylo_run(session, phylo_run_id)
+        group: Group = phylo_run.group
+
+        # Give the nexstrain config builder some info to make decisions
+        context = {
+            "num_sequences": num_sequences,
+            "num_included_samples": num_included_samples,
+        }
+        builder: BaseNextstrainConfigBuilder = builder_factory(
+            phylo_run.tree_type, group, phylo_run.template_args, **context
+        )
+        builder.write_file(builds_file_fh)
+
+        return f"YAML Build Config dumped to {builds_file_fh.name}"
```

Comment:
Nit: Since this is intended only ever to be printed/logged, I'd just `print` it here and have no return on the function. When I first read the cli block
```
    if dump_config:
        msg = dump_yaml_template(phylo_run_id, builds_file_fh)
        print(msg)
        return
```
I had assumed `msg` was actually the resulting YAML string, then getting printed up there. Since I can't see any other reason this returned value would be used than to print it, I think it makes more sense to just print it here and have the caller do nothing other than call the function.